evm/interchainTokenFactory.js

```diff
@@ -137,6 +137,9 @@ async function processCommand(config, chain, options) {
                 gasOptions,
             );
 
+            const tokenId = await interchainTokenFactory.interchainTokenId(wallet.address, deploymentSalt);
+            printInfo('tokenId', tokenId);
+
             await handleTx(tx, chain, interchainTokenService, options.action, 'TokenManagerDeployed', 'InterchainTokenDeploymentStarted');
```

Comment:
shouldn't `handleTx` check for `InterchainTokenDeployed` instead of `InterchainTokenDeploymentStarted` when not deploying remotely? I don't see the `InterchainTokenDeploymentStarted` event in a `deployInterchainToken` transaction (https://testnet.snowtrace.io/tx/0x0125c0d4006cc2d8fda1ae7c0c3f27507b2fdcc3ea90685b0c94b4b9006b1c2a/eventlog?chainid=43113) and I don't get the warning either